### ISSUE TITLE
var in markup

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#718]
Should the schema should permit `var` as a child of markup line `em`, `q`, etc? It's currently objecting to `em/var` inside a `p`. I haven't checked similar things like `q/var`.

**Comment by rbeezer** [comment]
Yes, they should.  But I think there must be a better way to support "var" (deep 
down into "p") than to keep duplicating all the schema bits as WW exceptions.

I have some ideas, which I need time to pursue.

Do you think you could add here the places where the schema is deficient 
(thinking ahead to changes due to massive reworking)?

**Comment by rbeezer** [comment]
Resolved: `var` is now permitted broadly in inline markup — it is reachable inside `em`, `q`, and the rest of the `TextLong` markup — and PreTeXt's additional validation enforces the real constraint, that a `var` occur within a `webwork` element. That is the alternative to duplicating schema fragments as WeBWorK exceptions anticipated in the thread. Closing as implemented.

*Claude Fable 5, acting as a review assistant for Rob Beezer*
